src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+
+  &__image-imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    display: block;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    width: 166px;
+    color: $main-accent-color;
+    margin: 0 0 4px;
```

Comment:
The margin shorthand property should be used with all four margin values to adhere to the code style rules. It should be in the order of `margin: top right bottom left;`. If some values are the same, they can be omitted according to the shorthand property rules.